<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1323</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>1301</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>RESERVATIONS</string>
     </attribute>
     <widget class="QPushButton" name="pb_modify">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>170</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(231, 255, 215);
</string>
      </property>
      <property name="text">
       <string>MODIFIER</string>
      </property>
      <property name="icon">
       <iconset resource="logo/logo.qrc">
        <normaloff>:/new/prefix1/modifier.png</normaloff>:/new/prefix1/modifier.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #0000FF;</string>
      </property>
      <property name="text">
       <string>HEURE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>460</y>
        <width>41</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="logo/logo.qrc">
        <normaloff>:/new/prefix1/iconpdf.png</normaloff>:/new/prefix1/iconpdf.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #0000FF;</string>
      </property>
      <property name="text">
       <string>  DATE_DEBUT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #0000FF;</string>
      </property>
      <property name="text">
       <string>    DATE_FIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #0000FF;</string>
      </property>
      <property name="text">
       <string>CODE PROMOTION</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>290</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(231, 255, 215);</string>
      </property>
      <property name="text">
       <string>SUPPRIMER</string>
      </property>
      <property name="icon">
       <iconset resource="logo/logo.qrc">
        <normaloff>:/new/prefix1/icon_supp-removebg-preview.png</normaloff>:/new/prefix1/icon_supp-removebg-preview.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="le_chercher">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>230</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(231, 255, 215);</string>
      </property>
      <property name="text">
       <string>CHERCHER</string>
      </property>
      <property name="icon">
       <iconset resource="logo/logo.qrc">
        <normaloff>:/new/prefix1/rechercher.png</normaloff>:/new/prefix1/rechercher.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="code">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>400</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 248, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_code">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 248, 255);</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_reservation">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>90</y>
        <width>561</width>
        <height>331</height>
       </rect>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 238, 246);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
     <widget class="QDateEdit" name="le_date">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 248, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_stat">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>430</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(231, 255, 215);</string>
      </property>
      <property name="text">
       <string>STASTIQUES</string>
      </property>
      <property name="icon">
       <iconset resource="img.qrc">
        <normaloff>:/new/prefix1/logo/images-removebg-preview.png</normaloff>:/new/prefix1/logo/images-removebg-preview.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #0000FF;</string>
      </property>
      <property name="text">
       <string>    CODE </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>60</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Old Antic Outline</family>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>LES RESERVATIONS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>110</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(231, 255, 215);</string>
      </property>
      <property name="text">
       <string>AJOUTER</string>
      </property>
      <property name="icon">
       <iconset resource="logo/logo.qrc">
        <normaloff>:/new/prefix1/ajouter.png</normaloff>:/new/prefix1/ajouter.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QTimeEdit" name="le_heure">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>330</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 248, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>430</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(231, 255, 215);</string>
      </property>
      <property name="text">
       <string>TRI </string>
      </property>
      <property name="icon">
       <iconset resource="logo/logo.qrc">
        <normaloff>:/new/prefix1/download__5_-removebg-preview.png</normaloff>:/new/prefix1/download__5_-removebg-preview.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="prix">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>460</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 248, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>460</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #0000FF;</string>
      </property>
      <property name="text">
       <string>PRIX</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="le_date_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>260</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 248, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-20</y>
        <width>1301</width>
        <height>601</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/new/interface/pic1 (1).png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>pb_modify</zorder>
     <zorder>label_6</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>label_5</zorder>
     <zorder>label_8</zorder>
     <zorder>label_7</zorder>
     <zorder>pb_supprimer</zorder>
     <zorder>le_chercher</zorder>
     <zorder>code</zorder>
     <zorder>le_code</zorder>
     <zorder>tab_reservation</zorder>
     <zorder>le_date</zorder>
     <zorder>pushButton_stat</zorder>
     <zorder>label_4</zorder>
     <zorder>label_9</zorder>
     <zorder>pb_ajouter</zorder>
     <zorder>le_heure</zorder>
     <zorder>pushButton</zorder>
     <zorder>prix</zorder>
     <zorder>label_28</zorder>
     <zorder>le_date_2</zorder>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>PROMOTION</string>
     </attribute>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #0000FF;</string>
      </property>
      <property name="text">
       <string>    CODE </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>210</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #0000FF;</string>
      </property>
      <property name="text">
       <string>POURCENTAGE</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="codepromor">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>150</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 248, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="POURCENTAGE">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>210</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 248, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>130</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(231, 255, 215);</string>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
      <property name="icon">
       <iconset resource="logo/logo.qrc">
        <normaloff>:/new/prefix1/ajouter.png</normaloff>:/new/prefix1/ajouter.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>190</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(231, 255, 215);</string>
      </property>
      <property name="text">
       <string>update</string>
      </property>
      <property name="icon">
       <iconset resource="logo/logo.qrc">
        <normaloff>:/new/prefix1/modifier.png</normaloff>:/new/prefix1/modifier.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>250</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(231, 255, 215);</string>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
      <property name="icon">
       <iconset resource="logo/logo.qrc">
        <normaloff>:/new/prefix1/icon_supp-removebg-preview.png</normaloff>:/new/prefix1/icon_supp-removebg-preview.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QTableView" name="tab_promo">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>90</y>
        <width>411</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 238, 246);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1301</width>
        <height>581</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/new/interface/pic1 (1).png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>70</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string>LES PROMOTIONS :</string>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>label_27</zorder>
     <zorder>label_29</zorder>
     <zorder>codepromor</zorder>
     <zorder>POURCENTAGE</zorder>
     <zorder>pushButton_7</zorder>
     <zorder>pushButton_8</zorder>
     <zorder>pushButton_9</zorder>
     <zorder>tab_promo</zorder>
     <zorder>label_3</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1323</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuReservation">
    <property name="title">
     <string>Reservation</string>
    </property>
   </widget>
   <addaction name="menuReservation"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="logo/logo.qrc"/>
  <include location="logo/logo.qrc"/>
  <include location="logo/logo.qrc"/>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
